docs/sources/flow/reference/components/prometheus.scrape.md

```diff
@@ -78,6 +78,7 @@ authorization | [authorization][] | Configure generic authorization to targets.
 oauth2 | [oauth2][] | Configure OAuth2 for authenticating to targets. | no
 oauth2 > tls_config | [tls_config][] | Configure TLS settings for connecting to targets via OAuth2. | no
 tls_config | [tls_config][] | Configure TLS settings for connecting to targets. | no
+clustering | [clustering][] | Configure the component for when the Agent is running in clustered mode | no
```

Comment:
Unrelated to this PR, but I was wondering if we have some convention that we use here - seems like we went with a block with a single bool attribute vs. having a bool attribute on the component itself?

Reply:
I suppose we will want to reuse the clustering block for other components in the future, @tpaschalis ?

Reply:
This could well be a single argument, but we opted it to be a block for two reasons

a) It will be easier to move to a single place to share with other components which implement clustering-specific behavior and add more fields to it.
b) The convention is that each block gets its own little markdown header/section so in this case it allowed us to expand on clustering with some more prose than if we had a single attribute (not that we couldn't do the same under the Attributes section, but it now stands more on its own)

Reply:
(sorry Paulin, missed your response at first, but yes ^^)